### PR TITLE
Add xorg-libxxf86vm

### PR DESCRIPTION
**Pull request by hmaarrfk**
needed for
* https://github.com/conda-forge/qt-main-feedstock/pull/114
* https://github.com/conda-forge/qt-main-feedstock/pull/115

cc: @pkgw would you like to co-maintain?

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
hmm. i kinda don't care about windows. This is really to support a specific usecase of qt on linux without CDTs. 

Is it ok to skip windows here?

File: recipes/xorg-libxxf86vm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - make                          # [unix]
+    - autoconf                      # [unix]
+    - automake                      # [unix]
+    - gettext                       # [unix]
+    - libtool                       # [unix]
+  host:
+    - xorg-util-macros
+    - xorg-xproto
+    - xorg-libx11
+    # Additional constraintless xorg-libx11 dep below is for conda-build to add potential pinning to.
```
Comment (pkgw) [inline recipes/xorg-libxxf86vm/meta.yaml:41]: The comment says "below" but the constraintless dep is "above", here.

**Comment by pkgw** [comment]
Sure, I'll help maintain.

I'm not sure whether libxxf86vm is supposed to be buildable on Windows, but it's certainly not one of the more cross-platform aspects of X.org. I think it's fine to skip the Windows build. I think we should keep the general Windows-friendly structure of the `build.sh` file to align with all of the other X.org packages, but we can add a header line pointing out that the Windows-y elements are untested/unused (but that we're keeping them because [what I just wrote]).

File: recipes/xorg-libxxf86vm/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipes/xorg-libxxf86vm/meta.yaml:45]: thanks.

**Comment by hmaarrfk** [comment]
I think i captured what you said in the comment.

**Pull request merged**